application/models/OtpDataBase.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class OtpDataBase {
+
+  protected $connection;
+
+  public function __construct() {
+    $this->connection = new mysqli(SERVER, USER, PASS, DBNAME);
+    // Check connection
+    if ($this->connection->connect_error) {
+      die("Connection failed: " . $conn->connect_error);
+    }
+  }
+
+  public function emailExist($id, $otp, $email) {
+    $sql = "select * from otp";
+    $result = $this->connection->query($sql);
+
+    if ($result->num_rows > 0) {
+      // output data of each row
+      while($row = $result->fetch_assoc()) {
+        $mailid[] = $row['email'];
```

Comment:
Same comment as emailExist() function in another class.
Write proper query to fetch required data instead of fetching and looping over data.